translations/ar.json

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
I might be wrong but we should only provide translation for english then Transifex would provide for the other languages

Reply:
What is Transifex? that would be cool but I remember even on patient-chart, we were manually adding the other language translations but if there is a tool to help in that, let's use it then. @donaldkibet do you know how we can use that Transifex?

Reply:
I am gonna go ahead and merge this until we see the Transifex option